qiskit/pulse/library/symbolic_pulses.py

```diff
@@ -147,19 +157,26 @@ class SymbolicPulse(Pulse):
     .. rubric:: Envelope function
 
     This is defined with an instance attribute :attr:`SymbolicPulse.envelope`
-    which can be provided through its setter method.
-    The expression must be a function of ``t``, ``duration``, ``amp`` and
-    any additional parameters specified for the pulse shape to implement.
-    The time ``t`` and ``duration`` are in units of dt, i.e. sample time resolution,
-    and this function is sampled with a discrete time vector in [0, ``duration``]
-    sampling the pulse envelope at every 0.5 dt (middle sampling strategy) when
-    :meth:`SymbolicPulse.get_waveform` method is called.
-    The ``amp`` is a complex-valued coefficient that scales the symbolic pulse envelope.
-    This indicates conventionally the Qiskit Pulse conforms to the IQ format rather
-    than the phasor representation. When a real value is assigned to the ``amp``,
+    which can be provided through the symbolic pulse constructor.
+    The envelope function at time :math:`t` must be defined in the form
+
+    .. math::
+
+        F(t, \Theta) = {\rm amp} \times F'(t, {\rm duration}, \overline{\rm params})
+
+    where :math:`\Theta` is the full pulse parameter in :attr:`SymbolicPulse.parameters`
+    dictionary which must include the :math:`\rm amp` and :math:`\rm duration`.
+    The :math:`\rm amp` is a complex value representing a scale factor and phase applied to
+    the envelope function :math:`F'` which is defined with :attr:`SymbolicPulse.envelope`.
```

Comment:
Would this be a good place to note that generally the envelope should have a maximum value of 1 so that `amp` denotes the maximum value of the pulse?

Reply:
Good point. I think it's difficult to assume envelope is normalized to 1 because it can be complex. For example, the real part of DRAG is always normalized, but there is finite increase of norm due to non-zero quadrature component.

Reply:
Right, it can't be a hard rule, just a suggestion for pulse shapes where it is feasible. That is why I suggested a `unit_scaling` attribute in my other comment so the pulse creator could mark such a pulse since it is not always true.

 (Technically, it's feasible for DRAG by rescaling the pulse but the expressions get very complex and beta > sigma starts redefining the amplitude; also, at least for beta < sigma, the DRAG envelope still has a maximum of 1).